### PR TITLE
feat: themes component to homepage

### PR DESCRIPTION
**Pull request by zalborzi**
1- Themes component created and added to the homepage
2- Search bar size
3- About page image fixed.

Screenshot 2024-07-16 at 15 13 55
Screenshot 2024-07-16 at 15 14 02
Screenshot 2024-07-16 at 15 14 12
Screenshot 2024-07-16 at 15 14 17
Screenshot 2024-07-16 at 15 14 37
Screenshot 2024-07-16 at 15 14 43
Screenshot 2024-07-16 at 15 23 04

## Summary by Sourcery

This pull request adds a new Themes component to the homepage, fixes an image display issue on the About page, and enhances the search bar layout for improved responsiveness.

- **New Features**:
    - Introduced a new Themes component to the homepage, displaying various themes with images and descriptions.
- **Bug Fixes**:
    - Fixed the image display issue on the About page.
- **Enhancements**:
    - Adjusted the search bar size and layout on the homepage for better responsiveness.

### CODE REVIEW

File: src/components/themesSection.tsx [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+  { name: "Transport", img: transportImg, desc: "Transport" },
+];
+
+const ThemesSection = () => {
+  return (
+    <section className="mb-20">
+      <div className="grid grid-cols-2 sm:grid-cols-3 md:grid-cols-4 lg:grid-cols-5 xl:grid-cols-6 gap-4">
+        {themes.map((theme) => (
+          <a
+            key={theme.name}
```
[the remark this answers is not in the document]
Comment (zalborzi) [reply src/components/themesSection.tsx]: never happens.

File: src/components/themesSection.tsx [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+              />
+              <h3 className="font-semibold text-lg sm:mt-1">{theme.name}</h3>
+              <div className="leading-4 max-w-[8.5rem]">{theme.desc}</div>
+            </div>
+          </a>
+        ))}
+      </div>
+      <div className="pt-4 px-8 sm:px-0 w-full flex">
+        <a
+          className="text-primary flex items-center gap-1 transition hover:underline duration-1000 ml-auto"
```
[the remark this answers is not in the document]
Comment (zalborzi) [reply src/components/themesSection.tsx]: increased to 500.

**Comment by zalborzi** [comment]
we can limit the shown on the main page by using NEXT_PUBLIC_THEMES_LIST

Screenshot 2024-07-16 at 15 55 38

**Review by brunopacheco1 — APPROVED** [review]

File: src/components/ThemesSection.tsx [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+    desc: "Justice, legal system and public safety",
+  },
+  { name: "Population", img: populationImg, desc: "Population and society" },
+  { name: "Regions", img: regionsImg, desc: "Regions and cities" },
+  { name: "Tech", img: techImg, desc: "Science and technology" },
+  { name: "Transport", img: transportImg },
+];
+
+const getThemesFromEnv = () => {
+  const themesEnv = process.env.NEXT_PUBLIC_THEMES_LIST;
```
[the remark this answers is not in the document]
Comment (sulejmank) [reply src/components/ThemesSection.tsx:57]: why we have them here hardcoded and also geting them from env? As they are only for LNDC I guess hardcoded should be enough?

[the remark this answers is not in the document]
Comment (zalborzi) [reply src/components/ThemesSection.tsx:57]: Dcat has been adopted by GDI, and the themes in Dcat AP3 will always remain consistent. No changes will ever be made to these 13 items, although some may be hidden as the topic is not as generic as LNDC.

**Pull request merged**